src/styles/blocks/main.scss

```diff
@@ -0,0 +1,11 @@
+.main {
```

Comment:
Better to use class name `.main` for `<main>` tag.
In this case, you could use smth like `.blocks`